packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -507,6 +507,37 @@ export class StripeHelper {
     return this.stripe.invoices.pay(invoice.id, { paid_out_of_band: true });
   }
 
+  /**
+   * Update the customer object to add customer's PayPal billing address.
+   * 
+   * @param customer_id 
+   * @param city 
+   * @param country 
+   * @param line1 
+   * @param line2 
+   * @param postal_code 
+   * @param state 
+   */
+  async updateCustomerBillingAddress(
+    customer_id: string,
+    city: string,
+    country: string,
+    line1: string,
+    line2: string,
+    postal_code: string,
```

Comment:
This should be camelCased.

Reply:
Hmm, I purposely did `postal_code` because it used that in here https://stripe.com/docs/api/customers/object#customer_object-address